src/errors.rs

```diff
@@ -171,6 +159,24 @@ fn on_config_error(error: ConfigError) -> ErrorMessage {
                             .call()
                     }
                 },
+
+                ParseConfigError::MissingNamespacedConfig => {
+                    create_error()
+                        .error_type(UserFacing(SuggestRetryBuild::Yes, SuggestSubmitIssue::No))
```

Comment:
Should this be `SuggestRetryBuild::No` instead? (This isn't a transient issue - they need to fix their config not just retry)